# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/Workarounds.h
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/Workarounds.h
--- a/gfx/angle/checkout/src/libANGLE/Workarounds.h	Fri Nov 08 16:41:42 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,34 +0,0 @@
-//
-// Copyright (c) 2016 The ANGLE Project Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-//
-
-// Workarounds.h: Workarounds for driver bugs and other behaviors seen
-// on all platforms.
-
-#ifndef LIBANGLE_WORKAROUNDS_H_
-#define LIBANGLE_WORKAROUNDS_H_
-
-namespace gl
-{
-
-struct Workarounds
-{
-    // Force the context to be lost (via KHR_robustness) if a GL_OUT_OF_MEMORY error occurs. The
-    // driver may be in an inconsistent state if this happens, and some users of ANGLE rely on this
-    // notification to prevent further execution.
-    bool loseContextOnOutOfMemory = false;
-
-    // Program binaries don't contain transform feedback varyings on Qualcomm GPUs.
-    // Work around this by disabling the program cache for programs with transform feedback.
-    bool disableProgramCachingForTransformFeedback = false;
-
-    // On Windows Intel OpenGL drivers TexImage sometimes seems to interact with the Framebuffer.
-    // Flaky crashes can occur unless we sync the Framebuffer bindings. The workaround is to add
-    // Framebuffer binding dirty bits to TexImage updates. See http://anglebug.com/2906
-    bool syncFramebufferBindingsOnTexImage = false;
-};
-}  // namespace gl
-
-#endif  // LIBANGLE_WORKAROUNDS_H_